docs/web3.eth.rst

```diff
@@ -987,6 +987,46 @@ The following methods are available on the ``web3.eth`` namespace.
     As of v6.3.0, the raw data is also returned and
     can be accessed via the ``data`` attribute on ``ContractLogicError``.
 
+
+.. py:method:: Eth.create_access_list(transaction, block_identifier=web3.eth.default_block)
+
+    * Delegates to ``eth_createAccessList`` RPC Method
+
+    This method creates an `EIP-2930 <https://eips.ethereum.org/EIPS/eip-2930>`_ type ``accessList`` based on a given
+    ``Transaction``. The ``accessList`` contains all storage slots and addresses read and written by the transaction,
```

Comment:
Sorry, another tiny nit we should've caught. This is implying the first param which we named `transaction`, lowercase, in the method definition. It should remain lowercase here.

```suggestion
    transaction. The ``accessList`` contains all storage slots and addresses read and written by the transaction,
```

Reply:
No trouble at all! Fixing this and then I'll merge.